### PR TITLE
142697163 seed db with test user and test portfolio

### PR DESCRIPTION
**Pull request by Laszlo-JFLMTCO**
@mollybrown @jdconrad89 @drod1000 
This PR covers [ticket ID 142697163](https://www.pivotaltracker.com/story/show/142697163) the following key changes:  
- Adds new rake task: `rake db:seed:test_user` to seed database with list of test users and test portfolios  
- Preserves original `Location` table implementation within `db/seeds.rb`  
- Adds database cleanup through `Location.delete_all` before seeding database

To confirm that new task is available, run `rake --task` in the terminal.

FYI @neight-allen

### CODE REVIEW

**Comment by drod1000** [comment]
@Laszlo-JFLMTCO Looks good, we can work off of this to build the one with associations

**Comment by jdconrad89** [comment]
@Laszlo-JFLMTCO  looks great! Thank you sir!

File: db/test_user.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    user1.create_portfolio(full_name: 'Leslie Hawk', email: 'example@example.com', github_url: 'github.com', linkedin_url: 'linkedin.com', bio: 'I like cars', title: 'Software Developer')
+    user2.create_portfolio(full_name: 'Joe Henry', email: 'example@example.com', github_url: 'github.com', linkedin_url: 'linkedin.com', bio: 'I like cars', title: 'Software Developer')
+    user3.create_portfolio(full_name: 'Jack Donley', email: 'example@example.com', github_url: 'github.com', linkedin_url: 'linkedin.com', bio: 'I like cars', title: 'Software Developer')
+    user4.create_portfolio(full_name: 'Greg Maker', email: 'example@example.com', github_url: 'github.com', linkedin_url: 'linkedin.com', bio: 'I like cars', title: 'Software Developer')
+    user5.create_portfolio(full_name: 'Sue Anne', email: 'example@example.com', github_url: 'github.com', linkedin_url: 'linkedin.com', bio: 'I like cars', title: 'Software Developer')
+    user6.create_portfolio(full_name: 'Sally May', email: 'example@example.com', github_url: 'github.com', linkedin_url: 'linkedin.com', bio: 'I like cars', title: 'Software Developer')
+    puts 'Finished adding test users and test portfolios...'
```
Comment (neight-allen) [inline db/test_user.rb:24]: Is there not a check for unique email? @Laszlo-JFLMTCO

Comment (jdconrad89) [reply db/test_user.rb:24]: @neight-allen just looked into the portfolio model. There currently is not uniqueness check for anything at the moment. I will open a card for that and get that fixed and tested.

Comment (neight-allen) [reply db/test_user.rb:24]: @jdconrad89 I'm not sure it's a priority, but I do think a card should be made so we don't forget.

**Pull request merged**